### PR TITLE
Formating fixes

### PR DESCRIPTION
**Pull request by dting**
@pdotsani

### CODE REVIEW

File: client/app/main/school-reports/school-reports.controller.js [hunk 1]
```diff
@@ -3,47 +3,61 @@
...
     exporterPdfPageSize: 'LETTER',
     exporterPdfMaxGridWidth: 500,
-    exporterPdfFooter: function ( currentPage, pageCount ) {
-      return { text: currentPage.toString() + ' of ' + pageCount.toString(), style: 'footerStyle' };
+    exporterPdfFooter: function(currentPage, pageCount) {
+      return {
+        text: currentPage.toString() + ' of ' + pageCount.toString(),
+        style: 'footerStyle'
+      };
     },
```
Comment (pdotsani) [inline client/app/main/school-reports/school-reports.controller.js]: I was thinking, maybe we should add the date of the report to the footer or header?

File: client/app/main/school-reports/school-reports.controller.js [hunk 2]
```diff
@@ -3,47 +3,61 @@
...
-    exporterPdfCustomFormatter: function ( docDefinition ) {
-      docDefinition.styles.headerStyle = { fontSize: 22, bold: true, color: '#265E6D' };
-      docDefinition.styles.footerStyle = { fontSize: 10, bold: true };
+    exporterPdfCustomFormatter: function(docDefinition) {
+      docDefinition.styles.headerStyle =
+      {fontSize: 22, bold: true, color: '#265E6D'};
+      docDefinition.styles.footerStyle = {fontSize: 10, bold: true};
       return docDefinition;
     }
```
Comment (pdotsani) [inline client/app/main/school-reports/school-reports.controller.js]: See above for footer and header.

File: client/components/outreach/outreach.service.js [hunk 3]
```diff
@@ -11,12 +11,6 @@ app.factory('Outreach', function($resource) {
       params: {
         controller: 'note'
       }
-    },
-    getTypes: {
-      method: 'GET',
-      params: {
-        controller: 'types'
-      }
     }
```
Comment (pdotsani) [inline client/components/outreach/outreach.service.js]: Nice, sorry forgot to take this out.

File: server/api/intervention/intervention.model.js [hunk 4]
```diff
@@ -3,8 +3,16 @@
...
+  'Letter Sent',
+  'Home Visit',
+  'SST Referral',
+  'Court Referral'
+];
+
 var InterventionSchema = new Schema({
-  type: {type: String, required: true},
+  type: {type: String, required: true, enum: types},
   tier: {type: Number, required: true},
```
Comment (pdotsani) [inline server/api/intervention/intervention.model.js]: Strange, I remember editing this on the last pull request. Maybe merge overrides?

**Comment by pdotsani** [comment]
lgtm. Just one suggestion regarding adding the date, could possibly do this to all pdf / csv exports.

**Comment by dting** [comment]
We can bring up the pdf exports options with Jennifer in the next meeting. I think that the pdf exports are probably sitting at a lower priority at this point.

**Pull request merged**